app/uk/gov/hmrc/customsexportsinternalfrontend/controllers/actions/Verify.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsexportsinternalfrontend.controllers.actions
+
+import javax.inject.Inject
+
+class Verify @Inject()(authenticatedAction: AuthenticatedAction) {
```

Comment:
What is the point of this class?

Reply:
I knew this would come up...
It's for readability in Controllers. 
We gradually end up with more actions over time, and i'd rather there was just one way to access them (through this class) rather than injecting many actions into each controller

Reply:
Low cohesion in objects is code smell. Same as injecting object just to access their fields.

Reply:
Removed